packages/playground/src/components/vm_deployment_table.vue

```diff
@@ -381,6 +374,10 @@ function updateItem(newItem: any) {
   }
 }
 
+function renameFlist(url: string) {
+  return url.substring(url.lastIndexOf("/") + 1, url.lastIndexOf("."));
```

Comment:
we agreed to show the last name in case it's an official image. but if it's a user image, we show the full name

Reply:
done.